obspy/io/stationtxt/core.py

```diff
@@ -248,10 +251,136 @@ def _read(obj):
         inv.networks.extend(list(networks.values()))
     else:
         # Cannot really happen - just a safety measure.
-        raise NotImplementedError("Unkown level: " + str(level))
+        raise NotImplementedError("Unknown level: " + str(level))
     return inv
 
 
+def _format_time(value):
+    if isinstance(value, UTCDateTime):
+        if value.microsecond == 0:
+            return value.strftime("%Y-%m-%dT%H:%M:%S")
+        else:
+            return value.strftime("%Y-%m-%dT%H:%M:%S.%f")
```

Comment:
For better or worse it's without the `Z` utc specifier in the FDSN spec, so we need to comply with that.